chill-algebird/src/main/scala/com/twitter/chill/algebird/AlgebirdSerializers.scala

```diff
@@ -95,3 +86,42 @@ class HLLMonoidSerializer extends KSerializer[HyperLogLogMonoid] {
     hllMonoids.getOrElseUpdate(bits, new HyperLogLogMonoid(bits))
   }
 }
+
+class SketchMapSerializer[K, V](implicit skmMonoid: SketchMapMonoid[K, V], valueMonoid: Monoid[V])
```

Comment:
okay, I had somehow overlooked the issue these cause. Sorry. Ideally there are no parameters to a kryo serializer, but that's not easy to do given how AdaptiveMatrix holds on to the `Monoid[V]` which was probably a design flaw.

Could we get by here at least only holding the valueMonoid? I'm not sure why you need skmMonoid too, since it looks like you are writing the rows and columns from the SketchMap instance itself.